### PR TITLE
[DO-1253] Fix the issue with inappropriate symbols via register

### PR DESCRIPTION
**Pull request by Andrii17s**
Done, but maybe in wrong way

### CODE REVIEW

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
wrong way.
1. Use regular expressions
2. Use RegexValidator from django.
3. RegexValidator can be connected to serializer field

File: users/serializers.py [hunk 1]
```diff
@@ -52,6 +54,13 @@ def validate(self, data):
                 err_msg_result = format_lazy('{err_msg} {v}.', err_msg=err_msg, v=v)
                 raise serializers.ValidationError(err_msg_result)
 
+        for field in ['first_name', 'last_name']:
+            value = data[field].lower()
+            x = re.findall("[^a-z0-9-.'а-яії]", value)
```
Comment (roman-tiukh) [inline users/serializers.py:59]: `А-ЯёЁЇїІіЄєҐґ`
maybe need to add here whitespace

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Comment by Andrii17s** [comment]
@roman-tiukh new version is ready

File: users/validators.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.core.validators import RegexValidator
+from django.utils.translation import ugettext_lazy as _
+
+
+validate_symbols = RegexValidator(
```
Comment (roman-tiukh) [inline users/validators.py:5]: name_symbols_validator ?

File: users/validators.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    regex=r"^[a-zA-Zа-яА-Я0-9 'іїёєґЄЇҐІЭ.`-]*$",
+    message=_("Only alphanumeric characters, digits, and '-. are allowed")
+)
+
+
+validate_triple = RegexValidator(
```
Comment (roman-tiukh) [inline users/validators.py:11]: two_in_row_validator ?

File: users/locale/uk/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -137,45 +137,49 @@ msgstr "У суперкористувача повинно бути встано
...
+#: users/models.py:53 users/models.py:114
 msgid "language"
 msgstr "мова"
 
-#: users/models.py:104
+#: users/models.py:93
+msgid "You have not paid the invoice"
+msgstr "У Вас не сплачений рахунок"
+
+#: users/models.py:110
```
Comment (roman-tiukh) [inline users/locale/uk/LC_MESSAGES/django.po:152]: ?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
image

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**